README.md

```diff
@@ -38,8 +38,98 @@ a port into a plugin for the IntelliJ Platform products, including PyCharm, WebS
 
 We suggest a couple of alternative ways of installation.
 
-**Bash and zsh completion scripts are provided** in completion/ directory,
-see [wiki for their installation instructions](https://github.com/VirtusLab/git-machete/wiki).
+### Bash
```

Comment:
Actually... I'd rather avoid putting all these details directly into README, esp. before Installation and Quick start sections which are far more important for the newcomers :/

Reply:
I rather thought about putting that manual into a separate file within this repository (as opposed to wiki, which is technically a separate repo) and updating the link in README... how about `completion/README.md`?

Reply:
yeah sure! Check now please.